test/spec/utils/Options.spec.ts

```diff
@@ -18,181 +18,168 @@ describe('n3Charts.Utils.Options', () => {
 
       expect(testing).to.equal(expected);
     });
-  });
 
-  describe('parse', () => {
-    beforeEach(() => {
+    it('should create a series property with the type array', () => {
       options = new Options();
-    });
-
-    it('should call parseSeries function with series property', () => {
-      var arg = { series: ['foo', 'bar'] };
-      var spy = sinon.spy(options, 'parseSeries');
 
-      options.parse(arg);
-
-      var testing = spy.calledWith(arg.series);
+      var testing = angular.isArray(options.series);
       var expected = true;
 
       expect(testing).to.equal(expected);
     });
 
-    it('should call parseSeries function with default arguments', () => {
-      var arg = { foo: 'bar' };
-      var spy = sinon.spy(options, 'parseSeries');
-
-      options.parse(arg);
+    it('should create a margin property with the type object', () => {
+      options = new Options();
 
-      var testing = spy.calledWith(Options.DEFAULT.series);
+      var testing = angular.isObject(options.margin);
       var expected = true;
 
       expect(testing).to.equal(expected);
```

Comment:
I was expecting a super cool tool now, like tests in YAML or sth :smile: 

Ok, I agree with you; I just have 2 more remarks that came to our mind. 
1) We will have loads of contributors, so the testing guidelines should be clear.
2) If we enforce `equal` and `eql`, you only need to look at the 2 variables to understand the test